docs/rackhd/configuration.rst

```diff
@@ -241,172 +243,13 @@ These parameters beging with *HTTP* and *HTTPS*.
-that was added to the **Sku Discovery graph** in the first step.
-Below is a snippet of the Discovery graph without the reboot task (the reboot task was originally located
-after the task 'catalog-lldp')
-
-.. code-block:: javascript
-
-   module.exports = {
-    friendlyName: 'Discovery',
-    injectableName: 'Graph.Discovery',
-    options: {
-        'bootstrap-ubuntu': {
-            'triggerGroup': 'bootstrap'
-        },
-        'finish-bootstrap-trigger': {
-            'triggerGroup': 'bootstrap'
-        }
-    },
-    tasks: [
-        {
-            label: 'bootstrap-ubuntu',
-            taskName: 'Task.Linux.Bootstrap.Ubuntu'
-        },
-        {
-            label: 'catalog-dmi',
-            taskName: 'Task.Catalog.dmi'
-        },
-        {
-            label: 'catalog-ohai',
-            taskName: 'Task.Catalog.ohai',
-            waitOn: {
-                'catalog-dmi': 'finished'
-            }
-        },
-        {
-            label: 'catalog-bmc',
-            taskName: 'Task.Catalog.bmc',
-            waitOn: {
-                'catalog-ohai': 'finished'
-            },
-            ignoreFailure: true
-        },
-        {
-            label: 'catalog-lsall',
-            taskName: 'Task.Catalog.lsall',
-            waitOn: {
-                'catalog-bmc': 'finished'
-            },
-            ignoreFailure: true
-        },
-        {
-            label: 'catalog-megaraid',
-            taskName: 'Task.Catalog.megaraid',
-            waitOn: {
-                'catalog-lsall': 'finished'
-            },
-            ignoreFailure: true
-        },
-        {
-            label: 'catalog-smart',
-            taskName: 'Task.Catalog.smart',
-            waitOn: {
-                'catalog-megaraid': 'finished'
-            },
-            ignoreFailure: true
-        },
-        {
-            label: 'catalog-driveid',
-            taskName: 'Task.Catalog.Drive.Id',
-            waitOn: {
-                'catalog-smart': 'finished'
-            },
-            ignoreFailure: true
-        },
-        {
-            label: 'catalog-lldp',
-            taskName: 'Task.Catalog.LLDP',
-            waitOn: {
-                'catalog-driveid': 'finished'
-            },
-            ignoreFailure: true
-        },
-       {
-            label: 'finish-bootstrap-trigger',
-            taskName: 'Task.Trigger.Send.Finish',
-            waitOn: {
-                'catalog-lldp': 'finished'
-            }
-        }
-    ]
-   };
-
-
-Once the above steps are completed (edited and saved) the service needs to be restarted:
+A node gets discovered and the BMC IPMI comes up with a default username/password. User can automatically set
+IPMI OBM settings  using a  default user name('__rackhd__') and an auto generated password in rackHD by adding the following
+to RackHD ``config.json``:
 
 .. code-block:: shell
 
-    sudo service on-taskgraph start
-
+   "obmOption": true  
```

Comment:
If you look at how the condition job is implemented, you will see the boolean type will not work well due to the boolean doesn't have a prototype function `toLowerCase`, you if you don't change the condition job code, at here, you need change `true` to `"true"`.